File: backend/Restaurant.API/Restaurant.Models/Score.cs
using System;
using System.Collections.Generic;

namespace Restaurant.Models;

public partial class Score
{
    public int score_id { get; set; }

    public int? rest_scoreid { get; set; }

    public int score { get; set; }

    public DateTime score_date { get; set; }

    public virtual Restaurants? RestScore { get; set; }
}

File: frontend/Restaurant.UI/src/app/models/score.ts
export class Score{
    score_id: number;
    rest_scoreid: number;
    score: number;
    score_date: Date;

    constructor (score_id: number, rest_scoreid: number, score: number, score_date: Date){
        this.score_id = score_id;
        this.rest_scoreid = rest_scoreid;
        this.score = score;
        this.score_date = score_date;
    }
}